app/services/projects_search/cross_core_searcher.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module ProjectsSearch
+
+  class CrossCoreSearcher < BaseSearcher
+    def options
+      ["no", "yes"]
+    end
+
+    def search(params)
+      if params == "yes"
+        cross_core_projects
+      else
+        single_facility_projects
+      end
+    end
+
+    def label_method
+      :humanize
+    end
+
+    def label
+      I18n.t("projects.index.cross_core_searcher.label")
+    end
+
+    def input_type
+      :select
+    end
+
+    private
+
+    def cross_core_projects
+      Projects::Project
+        .joins(:orders)
+        .where(orders: { facility_id: @current_facility_id })
```

Comment:
I think we have a scope that does this same thing:
```suggestion
      Projects::Project.for_facility(@current_facility_id )
```
... but isn't this logic going to include single facility projects as well?
for OrderDetail we have this scope:
```
  scope :cross_core, lambda {
    joins(order: [:facility, :cross_core_project])
    .where.not(orders: { cross_core_project_id: nil })
  }
```
... can you add something like that to the Project model?
then we can do:
`Projects::Project.cross_core.for_facility(@current_facility_id )`

Reply:
Oh, you're right! I added a case for specs but those single facility projects don't have any orders attached.